.babelrc

```diff
@@ -2,14 +2,21 @@
   "presets": [
     [
       "@babel/env",
-      {"targets": {"node":"current"}}
+      {
+        "targets": {
+          "node": "8",
+          "browsers": ">1%, not ie 11"
```

Comment:
:( no love for IE

Reply:
haha, just noticed that bigint also means we dropped support for safari haha

Reply:
rip. thats actually quite a big chunk or people